spacy_llm/tasks/templates/entity_linker.v1.jinja

```diff
@@ -0,0 +1,58 @@
+For each of the MENTIONS in the TEXT, resolve the MENTION to the correct entity listed in ENTITIES.
+Each of the ENTITIES is prefixed by its ENTITY ID. Each of the MENTIONS in the TEXT is surrounded by *.
+For each of the MENTIONS appearing in the text, output the ID of the description fitting them best.
+This ID has to be surrounded by single <>, for example <1>.
+Make sure you make a choice for each of the MENTIONS. Prefix the solution for each MENTION with "--- ".
```

Comment:
Shouldn't we also allow the LLM to output `<NIL>` if they find that no entities match?

Reply:
Continuing discussion in https://github.com/explosion/spacy-llm/pull/241#discussion_r1304473312.